app/controllers/comments_controller.rb

```diff
@@ -0,0 +1,18 @@
+class CommentsController < ApplicationController
+  def new
```

Comment:
The user should be able to create a comment on behalf of the currently logged-in user of the app.